spec/compiler/codegen/proc_spec.cr

```diff
@@ -847,4 +847,83 @@ describe "Code gen: proc" do
       end
     ))
   end
+
+  it "saves receiver value of proc pointer `->var.foo`" do
+    run(%(
+      class Foo
+        def initialize(@foo : Int32)
+        end
+
+        def foo=(@foo)
+        end
+
```

Comment:
```suggestion
        def foo=(@foo)
        end

```

Reply:
What is this change? Why do you insert empty line?

Reply:
Oh, I guess the suggestion didn't work.

These setters are never used in the tests and they can be removed.

Reply:
Thanks. I'll remove them.